### DISCUSSION TITLE
Union catalogue for e-resources using VuFind

### DISCUSSION THREAD

**Discussion by ankana1996** [vufind-org/vufind#2657] [Q&A]
Hi there!
I'm working in an academic library, and we are thinking of creating a union catalogue of subscribed e-resources of some libraries using VuFind. Any comments or suggestions about how to go about it would be of great help.
Also, will it be a good idea to manage the e-resources with the help of CORAL and then go for KOHA-CORAL integration, which can eventually be the option for integrating with VuFind?

**Comment by demiankatz** [comment]
@ankana1996, I'm not particularly familiar with CORAL or KOHA-CORAL, so I'm not sure exactly how this will work. With any VuFind integration, the first question is "what kind of data can you get out of the system?" -- once you know that, you can usually find a way to load it into VuFind and make use of it. So in this case, do you know what data KOHA-CORAL is able to export? Does OAI-PMH work with this data, for example?

**Comment by ankana1996** [comment]
Dear sir,
Thank you so much for your reply. 
As far as my knowledge KOHA is compatible with OAI-PMH. And we can integrate CORAL into KOHA to maintain ease in searching for printed resources and electronic ones in one go from OPAC. But, to be particular, I need to explore more to get confident about integrating KOHA-CORAL into VuFind.

**Reply by demiankatz** [reply]
Yes, Koha definitely supports OAI-PMH for its bibliographic records, and it's quite straightforward to integrate it with VuFind. I'm just not sure if the CORAL ERM records are exposed in the same way as bibliographic records (in which case it would be easy to  use them) or if they would have to be indexed using a different mechanism. I'll be interested to hear what you learn, and if you have questions/problems on the VuFind side, please let us know here!

**Comment by ankana1996** [comment]
Sure, sir. Will be happy to share. Thank you once again.
